lib/UtapiClient.js

```diff
@@ -145,6 +153,14 @@ class UtapiClient {
         return this;
     }
 
+    _isCounterEnabled(action) {
+        if (this.enabledOperationCounters.length > 0) {
```

Comment:
if `this.disabledOperationCounters` is set to true while the list for `this.enabledOperationCounters` is not empty, do we defer to the list of enabled op counters and ignore the `disableOperationCounters` setting?

Reply:
Yes, that's the idea 😄. I figured if anyone were setting `enabledOperationCounters` then they'd likely want that to take precedence. It's worth clarifying though, and I'll add a check in the CloudServer config to apply a constraint to make sure the user gets what they expect.

Reply:
cool, everything else lgtm :)